packages/eds-core-react/src/components/Banner/Banner.test.tsx

```diff
@@ -32,7 +33,38 @@ describe('Banner', () => {
     )
     expect(asFragment()).toMatchSnapshot()
   })
-
+  it('Should pass a11y test when only text', async () => {
+    const { container } = render(
+      <StyledBanner>
+        <Banner.Message>Banner message</Banner.Message>
+      </StyledBanner>,
+    )
+    expect(await axe(container)).toHaveNoViolations()
+  })
+  it('Should pass a11y test when text and icon', async () => {
+    const iconTestId = 'banner-icon-test'
+    const { container } = render(
+      <StyledBanner>
+        <Banner.Icon>
+          <Icon name="add" data-testid={iconTestId} />
```

Comment:
Is there something special with unused `data-testid` and `axe`? 😂

Reply:
copy paste 🐛🙈